django_mongodb/client.py

```diff
@@ -0,0 +1,21 @@
+import signal
+
+from django.db.backends.base.client import BaseDatabaseClient
+
+
+class DatabaseClient(BaseDatabaseClient):
+    executable_name = "mongo"
+
+    @classmethod
+    def settings_to_cmd_args_env(cls, settings_dict, parameters):
+        raise NotImplementedError
```

Comment:
Let's raise an issue for this ticket as well. Not sure to what extent this will get used, but I do see this as the proxy for making any command line calls.

Reply:
It is https://github.com/mongodb-labs/django-mongodb/issues/3. During our last call, I suggested this as a good starter task for @WaVEV.